src/src/components/Contexts/CurrentViewContext.tsx

```diff
@@ -65,6 +65,15 @@ const commonPrefix = `/calendar/u/0/r/`;
  * Extract current date from the Google Calendar URL
  */
 function parsePath(path: string): CurrentView | undefined {
+  if (path === '/calendar/u/0/r') {
+    // There is possibly more nuance to this, but I am not 100% sure. I think a sufficient fix for now
+    const today = new Date();
+    return {
+      view: 'day',
```

Comment:
Uhgg... I have just discovered something bad.

The `/calendar/u/0/r` URL does not correspond to the day view, it uses the view that the user used last.
Thus, you will have to detect what view was rendered by Google Calendar in some way (either from the current value of the View switcher button or from the content)

Tell me if you would need help with that

Reply:
Or if we need a short-term solution, treating `/calendar/u/0/r` as a week view might be good enough (as week view is the default first time you open google calendar)

Reply:
I see. I will look into getting the view type, and if I can't get that then maybe we settle for the week view

Reply:
This should be resolved by the new push